new-lamassu-admin/src/components/LogsDownloaderPopper.js

```diff
@@ -121,53 +121,71 @@ const styles = {
 
 const useStyles = makeStyles(styles)
 
-const LogsDownloaderPopover = ({ id, open, anchorEl, onClose, logsResponse, ...props }) => {
-  const [radioButtons, setRadioButtons] = useState(0)
+const LogsDownloaderPopover = ({ id, name, open, anchorEl, getTimestamp, logs, title, ...props }) => {
+  const radioButtonOptions = [{ label: 'All logs', value: 'all' }, { label: 'Date range', value: 'range' }]
+
+  const [selectedRadio, setSelectedRadio] = useState(radioButtonOptions[0].value)
   const [range, setRange] = useState(null)
 
   const classes = useStyles()
 
   const dateRangePickerClasses = {
-    [classes.dateRangePickerShowing]: radioButtons === 1,
-    [classes.dateRangePickerHidden]: radioButtons === 0
-  }
-
-  const formatDateFile = date => {
-    return moment(date).format('YYYY-MM-DD_HH-mm')
+    [classes.dateRangePickerShowing]: selectedRadio === radioButtonOptions[1].value,
+    [classes.dateRangePickerHidden]: selectedRadio === radioButtonOptions[0].value
   }
 
   const handleRadioButtons = (event) => {
-    setRadioButtons(toInteger(event.target.value))
+    const radio = event.target.value
+    setSelectedRadio(radio)
   }
```

Comment:
this can be reduced to `const handleRadioButtons = compose(setSelectedRadio, get('target.value'))`